department-of-veterans-affairs/caseflow
# Moving program/levels/name into description

Connects #4024 

To test, do each of the following on UAT:
1) Load a worksheet with issues that are already saved in the database and confirm they're displayed correctly
2) Load a worksheet with issues that are not already saved in the database and confirm they're displayed correctly.
